src/Cli/dotnet/commands/dotnet-restore/RestoreCommandParser.cs

```diff
@@ -83,6 +84,11 @@ private static Option[] ImplicitRestoreOptions(bool showHelp = false, bool useSh
                           .WithSuggestionsFrom(_ => Suggest.RunTimesFromProjectFile())
                           .With(name: LocalizableStrings.CmdRuntimeOption)
                           .ForwardAsSingle(o => $"-property:RuntimeIdentifiers={string.Join("%3B", o.Arguments)}")),
+                Create.Option(
```

Comment:
Should this be using `CommonOptions.CurrentRuntimeOption`?

Reply:
Sure, pushed the update. I thought I did it for consistency; but now looking with (relatively) fresh eyes, the reason why `--rutime` is explicitly redefined here might be because MSBuild properties is pluralized. :)